lldb/cmake/modules/LLDBConfig.cmake

```diff
@@ -70,6 +70,9 @@ option(LLDB_SKIP_STRIP "Whether to skip stripping of binaries when installing ll
 option(LLDB_ENABLE_SWIFT_SUPPORT "Enable swift support" ON)
 option(LLDB_USE_STATIC_BINDINGS "Use the static Python bindings." OFF)
 option(LLDB_ENABLE_WERROR "Fail and stop if a warning is triggered." ${LLVM_ENABLE_WERROR})
+if(LLDB_ENABLE_SWIFT_SUPPORT)
+  add_definitions( -DLLDB_ENABLE_SWIFT )
+endif()
 # END SWIFT CODE
```

Comment:
Should all CMake code also just be wrapped inside `if(LLDB_ENABLE_SWIFT_SUPPORT)` or do we need extra code to set that up? I.e., can we get rid of (most) `# BEGIN SWIFT CODE` comments?

Reply:
All code that breaks normal compilation without Swift should be wrapped in an if/ifdef, the remaining code should be inside the `SWIFT CODE` comments until it's upstreamed (so that the LLDB we compile from this repo is as similar to LLDB+Swift as possible). I already put all the CMake code that prevents compilation behind the `if(LLDB_ENABLE_SWIFT_SUPPORT)`. The remaining code that adds Swift support to the test suite can also be wrapped, but that will be a follow up PR. There is also the SWIG binding code, but that's not really Swift specific so that stays with comments until I have a better idea for this.